core/src/main/resources/db/changelog/changesets/2024_06_12_family_linkage_modified.yaml

```diff
@@ -0,0 +1,26 @@
+databaseChangeLog:
+  - changeSet:
+      id: "multiple_family_linkage"
+      author: connorlbark
+      changes:
+        - dropColumn:
+            tableName: enrollee
+            columnName: family_id
+        - createTable:
+            tableName: family_member
```

Comment:
Our convention is to have join tables be named as boringly as possible.  let's call this family_enrollee, or enrollee_family

Reply:
I found `FamilyEnrollee`  slightly confusing because it sounds like it is `a family that is signed up as an enrollee` and `EnrolleeFamily` sounds like it is the actual `Family` object but I see your point about following convention, so it is probably better to follow it